packages/dai/src/index.js

```diff
@@ -1,14 +1,14 @@
 import Maker from './Maker';
 import { currencies } from './eth/Currency';
 import { stringToBytes32 } from './utils/conversion';
-import QueryApi from './QueryApi';
```

Comment:
Noticed a missing default export from `./QueryApi` here. Not sure if I'm missing something 🤔